### ISSUE TITLE
Add support for email attachments when there is no Mail.app account configured

### ISSUE THREAD

**Issue by natan** [square/Aardvark#47]
Currently if a customer does not have a Mail.app account configured, Aardvark's email reporter does not include attachments like logs and screenshots (which are **so** useful). For the common case of Gmail accounts, Aardvark could utilize [Google's iOS library](https://developers.google.com/gmail/api/quickstart/ios?ver=swift) to send attachments directly from Aardvark.

**Comment by dfed** [comment]
I'd suggest adding a subpod with an EmailReporter subclass that has more fleshed-out Google support (and the Google library dependency) rather than shoving increased binary size onto all customers.

Thoughts on that approach? Not sure what the best binary size / simplicity of adoption trade-off is.

**Comment by natan** [comment]
Good point! This guide seems to go through the same exercise, with recommendations: http://www.dbotha.com/2014/12/04/optional-cocoapod-dependencies/

**Comment by NickEntin** [comment]
cc @EricMuller22
